projects/core/src/i18n/i18next/i18next-translation.service.ts

```diff
@@ -1,19 +1,36 @@
-import { Injectable, isDevMode } from '@angular/core';
-import i18next from 'i18next';
+import { Inject, Injectable, isDevMode } from '@angular/core';
+import i18next_global_instance, { i18n } from 'i18next';
 import { Observable } from 'rxjs';
 import { I18nConfig } from '../config/i18n-config';
 import { TranslationChunkService } from '../translation-chunk.service';
 import { TranslationService } from '../translation.service';
+import { I18NEXT_INSTANCE } from './i18next-instance';
 
 @Injectable({ providedIn: 'root' })
 export class I18nextTranslationService implements TranslationService {
   private readonly NON_BREAKING_SPACE = String.fromCharCode(160);
   protected readonly NAMESPACE_SEPARATOR = ':';
 
+  /**
+   * @deprecated use instead the constructor variant with `I18NEXT_INSTANCE` token as the 3rd parameter.
+   */
+  constructor(config: I18nConfig, translationChunk: TranslationChunkService);
```

Comment:
Please, double check if you are getting the new i18next parameter properly injected in the prod build, if this overload is specified as a first one.